File: unix/boot/spp/rpp/ratlibf/fold.f
      subroutine fold (token)
      integer token (100)
      integer clower
      integer i
      i = 1
23000 if (.not.(token (i) .ne. -2))goto 23002
      token (i) = clower (token (i))
23001 i = i + 1
      goto 23000
23002 continue
      return
      end

File: unix/boot/spp/rpp/ratlibr/fold.r
include	defs

# fold - fold all letters in a string to lower case

   subroutine fold (token)
   character token (ARB)

   character clower

   integer i

   for (i = 1; token (i) != EOS; i = i + 1)
      token (i) = clower (token (i))

   return
   end
